components/success.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ComponentPropsWithoutRef } from 'react';
+import Image from 'next/image';
+
+import { cn } from '@/lib/utils';
+import { ubuntu } from '@/app/fonts';
+
+type SuccessProps = ComponentPropsWithoutRef<'div'>;
+
+export const Success = ({ className, ...props }: SuccessProps) => {
+  return (
+    <div
+      className={cn(
+        'flex w-full flex-col items-center justify-center gap-4',
+        className,
+      )}
+      {...props}
+    >
+      <Image
+        src="/confetti-minimalistic.png"
+        alt="Confetti"
+        width={42}
+        height={42}
+      />
+      <p
```

Comment:
use svg instead of png 
![image](https://github.com/user-attachments/assets/debab30d-ea6f-4466-ac9f-e34136a9f69b)